src/makeCalculator.js

```diff
@@ -4,7 +4,47 @@
  * @return {object}
  */
 function makeCalculator() {
```

Comment:
According to the [NAMING] rule, the function `makeCalculator` should return an object named `calculator`, which is correctly implemented here.